toolchain.mk

```diff
@@ -35,7 +41,7 @@ define dltc
 endef
 
 .PHONY: toolchains
-toolchains: aarch32 aarch64 aarch64-legacy
+toolchains: aarch32 aarch64 aarch64-legacy aarch32-legacy
```

Comment:
Quick note: IMO it would be better to download only the necessary toolchain(s), either `aarch32` or `aarch32-legacy` depending on `COMPILE_LEGACY`. But this should be left for a separate PR, especially since we're already downloading both 32- and 64-bit toolchains when QEMU (v7) although the 64-bit compiler will never get used.

Reply:
Agree with that.